### DISCUSSION TITLE
FAQ ssl_show_warn needs to be updated

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#762] [Ideas]
current

```
Currently ElastAlert 2 uses elasticserarch-py 7.0.0, but the target parameters are the parameters that can be used from elasticserarch-py 7.5.0. In the future, we will end support for elasticsearch in the past and raise elasticserarch-py to 7.5.0 or later.
```

update

```
Now supported as of ElastAlert 2.4.0.
```

**Comment by jertel** [comment]
And this one, also in the FAQ:
![image](https://user-images.githubusercontent.com/16639611/157311303-71bf056a-07a3-42a0-8bcd-de1be2af06ef.png)

**Reply by nsano-rururu** [reply]
It requires a program fix.

**Reply by nsano-rururu** [reply]
You have also set up a link, but you haven't seen it.
https://elastalert2.readthedocs.io/en/latest/recipes/faq.html#i-ve-set-ssl-show-warn-but-it-doesn-t-seem-to-work

**Reply by nsano-rururu** [reply]
![キャプチャ](https://user-images.githubusercontent.com/22293449/157339989-0b1b99a8-960b-429e-8025-67941a850daa.PNG)

**Reply by nsano-rururu** [reply]
I submitted a pull request
https://github.com/jertel/elastalert2/pull/764
